qutebrowser/browser/webelem.py

```diff
@@ -50,6 +50,7 @@
     Group.url: '[src], [href]',
     Group.prevnext: 'a, area, button, link, [role=button]',
     Group.focus: '*:focus',
+    Group.inputs: 'input[type=text], input[type=password], textarea',
```

Comment:
Yeah, I think it makes sense to keep it. It just isn't listed in `_is_editable_input` because that only handles `input` elements. It seems you didn't push your update yet, though?